_maps/RandomRuins/IceRuins/icemoon_underground_hermit.dmm

```diff
@@ -0,0 +1,452 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"bz" = (
+/obj/item/flashlight/lantern,
+/turf/open/floor/grass/fairy,
+/area/ruin/powered/shuttle)
+"cI" = (
+/turf/open/floor/plating,
+/area/ruin/powered/shuttle)
+"de" = (
+/obj/item/flashlight/lantern,
+/turf/open/floor/plating/asteroid/snow/icemoon,
+/area/icemoon/underground/explored)
+"ec" = (
+/obj/item/clothing/suit/hooded/wintercoat,
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"ei" = (
+/obj/machinery/hydroponics/soil,
+/turf/open/floor/grass/fairy,
+/area/ruin/powered/shuttle)
+"gr" = (
+/turf/open/floor/grass/fairy,
+/area/ruin/powered/shuttle)
+"ha" = (
+/obj/machinery/light{
+	dir = 8
+	},
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"hv" = (
+/obj/structure/bonfire/prelit,
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"hN" = (
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"ii" = (
+/obj/effect/mob_spawn/human/hermit{
+	flavour_text = "You've been living on the ice moon for at least 40 years now! You've survived the ice whelps, hunted those damn bears and tamed a few wolves. What do those outsiders want?";
+	mob_name = "a grumy old man";
+	name = "Cryostasis bed";
+	short_desc = "You've been hunting polar bears for 40 years now! What do these 'NaniteTrans' newcomers want?"
+	},
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"lH" = (
+/obj/structure/barricade/wooden/crude/snow,
+/turf/open/floor/plating/asteroid/snow/icemoon,
+/area/icemoon/underground/explored)
+"mN" = (
+/turf/closed/wall/mineral/wood,
+/area/ruin/powered/shuttle)
+"oJ" = (
+/obj/structure/sink,
+/turf/open/floor/plating,
+/area/ruin/powered/shuttle)
+"pI" = (
+/obj/item/chair/wood/wings,
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"sC" = (
+/obj/item/gun/ballistic/rifle/boltaction,
```

Comment:
does the hermit get any ammo for this thing?

Reply:
It spawns with a full clip. Any more ammo and I fear it'd just become a role that serves purely to gun down miners.